.github/workflows/standards-and-tests.yml

```diff
@@ -15,25 +15,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.4]
+        php: [7.4, 8.0]
         os: [ubuntu-20.04]
-        epubcheck: [4.2.4]
-        prince: [14.2-1]
-        wordpress: [6.0.2]
-        experimental: [false]
+        epubcheck: [4.2.6]
+        prince: [14.3-1]
+        wordpress: [6.0.2, latest]
         include:
-          - php: 7.4
-            os: ubuntu-20.04
-            wordpress: latest
-            epubcheck: 4.2.6
-            prince: 14.2-1
-            experimental: true
-          - php: 8.0
-            os: ubuntu-20.04
-            wordpress: latest
-            epubcheck: 4.2.6
-            prince: 14.3-1
-            experimental: true
+          - experimental: true
+          - experimental: false
+            php: 8.0
+            wordpress: 6.0.2
+
```

Comment:
I've updated the pipeline a bit so now it runs the following matrix

|. PHP | WP | experimental |
| :-: | :-: | :-: |
| 7.4 | 6.0.2 | true |
| 7.4 | latest | true |
| 8.0 | 6.0.2 | false |
| 8.0 | latest | true |

This results on 4 different jobs that will run tests, however only the PHP 8.0 + WP 6.0.2 will run the CS and Code coverage. In the future when we drop PHP 7.4 support we simply remove 7.4 from the test matrix and the pipeline should be ok.